src/components/ChatEntry.js

```diff
@@ -1,22 +1,54 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
 
 const ChatEntry = (props) => {
+  const id = props.id
+  const sender = props.sender;
+  const body = props.body;
+  const liked = props.liked;
+  const updateChatLog = props.updateChatLog;
+  const timeStamp = props.timeStamp
+
+  const determineSender = (sender) => {
+    if (sender === 'Vladimir'){
+      const senderType = true;
+      return (senderType ? 'remote': 'local');
```

Comment:
This can just be simplified to `return 'remote'` and the else can just be `return 'local'`. Great job taking on this optional enhancement!